### DISCUSSION TITLE
Angular

### DISCUSSION THREAD

**Discussion by gloufy** [shakacode/shakapacker#475] [Q&A] [answered]
Hi everybody I didn't manage to use shakapacker with angular 13 Some body did success to do that ?

I've this error : 
This constructor is not compatible with Angular Dependency Injection because its dependency at index 0 of the parameter list is invalid

Thank for your help !

**Comment by tomdracz** [comment, marked as answer]
This seems to be a problem with your code rather than Shakapacker itself and could be something like https://stackoverflow.com/questions/60149052/this-constructor-is-not-compatible-with-angular-dependency-injection-because-its


If you think it's a Shakapacker issue, please provide a reproduction code/repository that we could run and test.
